onecgiar-pr-client/src/app/shared/guards/check-admin.guard.ts

```diff
@@ -1,13 +1,13 @@
 import { Injectable } from '@angular/core';
-import { ActivatedRouteSnapshot, CanActivate, RouterStateSnapshot, UrlTree, Router } from '@angular/router';
+import { ActivatedRouteSnapshot, RouterStateSnapshot, UrlTree, Router } from '@angular/router';
 import { Observable } from 'rxjs';
 import { RolesService } from '../services/global/roles.service';
 import { AuthService } from '../services/api/auth.service';
 
 @Injectable({
   providedIn: 'root'
 })
-export class CheckAdminGuard implements CanActivate {
```

Comment:
@xKeCo shouldn't all guards implement the CanActivate interface? why was it removed?

Reply:
@gcmartinezg CanActivate was automatically removed by Angular CLI during the update to newer versions, as it had been marked as deprecated.